app/models/journal_entry.rb

```diff
@@ -1,5 +1,6 @@
 class JournalEntry < ApplicationRecord
   belongs_to :user
+  has_one_attached :attached_document
```

Comment:
Could you rename it to just `document`?

Reply:
Also let's add a validation for it's acceptable content types

Reply:
it is about ataching documents to a journal entry so i believe this name describes it pretty well, regarding validating content types i will ad a story in my backlog, but Due to the deadline in 2 days i wont add this in this PR, but thank you for the recomendation.